docs/aquacomputer-octo-guide.md

```diff
@@ -58,6 +62,8 @@ Aquacomputer Octo (experimental)
 └── Fan 8 current     0.00  A
 ```
 
+_Changed in 1.12.0: read virtual temperature sensors as well._  
```

Comment:
FYI, in the future: I just now **changed** and [documented](https://github.com/liquidctl/liquidctl/blob/main/docs/developer/process.md#newchanged-in-notes) how new notes will look:

```markdown
_Changed in git: read virtual temperature sensors as well._<br>
```

Reply:
Woops, sorry :( Missed that.